common/appointment/create.ts

```diff
@@ -208,11 +208,14 @@ const createZoomMeetingForAppointmentWithHosts = async (
 export const saveZoomMeetingReport = async (appointment: Lecture) => {
     const result = await getZoomMeetingReport(appointment.zoomMeetingId);
 
+    if (!result) {
+        logger.info(`Meeting report could not be saved for appointment (${appointment.id})`);
```

Comment:
What about an early return here or does it make sense to still try to save the null value in the DB?